### PR TITLE
Make orchestrator command Windows aware

### LINKED ISSUE #3807: Make orchestrators command Windows-aware
Currently the orchestrators command returns the list of valid Kubernetes versions for Linux only which is not always equal to the list of valid Kubernetes versions for a cluster with Windows nodes. We should add an option to return the versions for Windows clusters such as: 
```
acs-engine orchestrators --orchestrator kubernetes --windows
```

### PR DESCRIPTION
**Pull request by sylus**
**What this PR does / why we need it**: Make orchestrators command Windows-aware

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #3807

**Special notes for your reviewer**: Was unsure whether to update `GetOrchestratorVersionProfileListV20170930` as well.

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/orchestrators.go [hunk 1]
```diff
@@ -35,12 +36,13 @@ func newOrchestratorsCmd() *cobra.Command {
 	f := command.Flags()
 	f.StringVar(&oc.orchestrator, "orchestrator", "", "orchestrator name (optional) ")
 	f.StringVar(&oc.version, "version", "", "orchestrator version (optional)")
+	f.BoolVar(&oc.windows, "windows", false, "orchestrator platform (optional)")
```
Comment (CecileRobertMichon) [inline cmd/orchestrators.go]: Could you add a note here that the windows flag applies to the Kubernetes orchestrator only?

For example: `orchestrator platform (optional, applies to Kubernetes only)`

File: pkg/api/orchestrators.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func GetOrchestratorVersionProfileListVLabs(orchestrator, version string) (*vlab
 
 // GetOrchestratorVersionProfileListV20170930 returns v20170930 OrchestratorVersionProfileList object per (optionally) specified orchestrator and version
 func GetOrchestratorVersionProfileListV20170930(orchestrator, version string) (*v20170930.OrchestratorVersionProfileList, error) {
-	apiOrchs, err := getOrchestratorVersionProfileList(orchestrator, version)
+	apiOrchs, err := getOrchestratorVersionProfileList(orchestrator, version, false)
```
Comment (CecileRobertMichon) [inline pkg/api/orchestrators.go]: I think that's fine to keep current behavior as-is

**Comment by tariq1890** [comment]
Thanks @sylus ! Looks good, but can we add unit tests for this?

**Comment by tariq1890** [comment]
I would really appreciate some unit tests for this.

**Comment by sylus** [comment]
Hey I am going to take a look at that this weekend, is my first time so might take a few days.

**Pull request merged**